apis/v1alpha1/rhoamtenant_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const ()
+
+// RhoamTenantSpec defines the desired state of RhoamTenant
+type RhoamTenantSpec struct {
+	Username string `json:"username,omitempty"`
```

Comment:
See comment below on just using the CR name

Reply:
Done